### ISSUE TITLE
Can't connect the wallet to any dApps in Opera browser

### ISSUE THREAD

**Issue by malyok** [RabbyHub/Rabby#1814]
Hello team. I can't connect Rabby wallet to any dApps on Opera browser. I faced with it a week ago. On Chrome the extension is working correctly.

The following error I see in the console:
```
Cannot set Core window.ethereum provider TypeError: Cannot redefine property: ethereum
    at Function.defineProperty (<anonymous>)
    at inpage.js:4:89246
    at inpage.js:4:89582
    at m (inpage.js:4:89866)
    at inpage.js:4:89969
    at inpage.js:4:89975
    at inpage.js:4:89979
(anonymous) @ inpage.js:4
```

Could you take a look, please?

Thanks in advance!

**Comment by RylaiCF** [comment]
have the same issue, please help

**Comment by vvvvvv1vvvvvv** [comment]
fixed in #1815
